internal/rbd/rbd_util.go

```diff
@@ -1145,9 +1141,11 @@ func generateVolumeFromVolumeID(
 	return rbdVol, err
 }
 
-// genVolFromVolID generates a rbdVolume structure from the provided identifier, updating
+// GenVolFromVolID generates a rbdVolume structure from the provided identifier, updating
 // the structure with elements from on-disk image metadata as well.
-func genVolFromVolID(
+//
+// nolint // returns non-exported *rbdVolume, which is fine
+func GenVolFromVolID(
```

Comment:
this is exported because you want to use this in csi-addons code?

Reply:
Yes! I want to use this in `internal/csi-addons/rbd` and with the `driver.go` in the `rbd` package the circular import is happening (not allowed by Go).